src/esp/assets/ResourceManager.cpp

```diff
@@ -170,6 +170,57 @@ bool ResourceManager::loadScene(
   return meshSuccess;
 }
 
+void ResourceManager::initPhysicsManager(
+    std::shared_ptr<physics::PhysicsManager>& physicsManager,
+    PhysicsManagerAttributes::ptr physicsManagerAttributes) {
+  //! PHYSICS INIT: Use the above config to initialize physics engine
+  bool defaultToNoneSimulator = true;
+  if (physicsManagerAttributes->getSimulator().compare("bullet") == 0) {
+#ifdef ESP_BUILD_WITH_BULLET
+    physicsManager.reset(
+        new physics::BulletPhysicsManager(*this, physicsManagerAttributes));
```

Comment:
Maybe pass in the PhysicsSceneAttributes also. Maybe add keys for the scene attributes to physicsManagerAttributes? We can refactor later for multi-scene support.

Reply:
How about we pass a reference to the PhysicsSceneAttributes map?  We can then specify which scene via passing the key to that scene.  This will support future multi-scenes more easily with minimal modifications.